table/policy.go

```diff
@@ -3453,6 +3454,13 @@ func (r *RoutingPolicy) GetDefinedSet(typ DefinedType, name string) (*config.Def
 	if !ok {
 		return nil, fmt.Errorf("invalid defined-set type: %d", typ)
 	}
+
```

Comment:
Again - unlock after copying the values over to `dl`

Reply:
Thanks, I'll unlock it after reading it.